spec/rspec/mocks/verifying_doubles/naming_spec.rb

```diff
@@ -45,37 +45,37 @@ def error_message_for(_)
 
       describe "instance_double" do
         it_behaves_like "a named verifying double", "InstanceDouble" do
-          alias :create_double :instance_double
+          alias_method :create_double, :instance_double
```

Comment:
This is something RuboCop insisted on. And there were JRuby failures that made me think might be related do this.
I don't mind reverting this and shutting up the cop.

Reply:
I do not mind this change